### PR TITLE
Update information-for-dns-records.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the steps to add DNS records

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4867

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/get-help-with-domains/information-for-dns-records.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ description: "Learn to find the values/information you need to create DNS record
     
 2. On the **Domains** page, select your domain. 
     
-3. Under **Required DNS settings**, you'll see the DNS records to add.
+3. Choose  **Manage DNS**, Click on **More Options**, select **Add your own DNS** and cick on **Continue** to see the DNS records to add.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/information-for-dns-records.md]:
```suggestion
3. Choose  **Manage DNS**, select **More Options** > **Add your own DNS** and select **Continue** to see the DNS records to add.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@scotvidican Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@scotvidican Gentle reminder.

**Review by cmcatee-MSFT — CHANGES_REQUESTED** [review]
Please change **Setup** to **Settings** at line 68, 74, and 80.

File: microsoft-365/admin/get-help-with-domains/information-for-dns-records.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ description: "Learn to find the values/information you need to create DNS record
     
 2. On the **Domains** page, select your domain. 
     
-3. Under **Required DNS settings**, you'll see the DNS records to add.
+3. Choose  **Manage DNS**, select **More Options** > **Add your own DNS** and select **Continue** to see the DNS records to add.
```
Comment (cmcatee-MSFT) [inline microsoft-365/admin/get-help-with-domains/information-for-dns-records.md:86]: Please send screenshots for steps 1 through 3 to me at cmcatee@microsoft.com. I'm not seeing the same UI.

**Comment by msbemba** [comment]
> Please change **Setup** to **Settings** at line 68, 74, and 80.

I have made those changes now. Please review and merge.

**Review by cmcatee-MSFT — CHANGES_REQUESTED** [review]
For lines 92-94 (Steps 4 & 5), please replace with the following:

`4. Go to [Add DNS records to connect your domain](create-dns-records-at-any-dns-hosting-provider.md), and follow the steps to add records at your DNS host's website.'

File: microsoft-365/admin/get-help-with-domains/information-for-dns-records.md (hunk 2 above)
Comment (msbemba) [reply microsoft-365/admin/get-help-with-domains/information-for-dns-records.md:86]: @cmcatee-MSFT  Looks like the options have changed in the last 25 days. I shall update you once I make these changes to the document.

Comment (msbemba) [reply microsoft-365/admin/get-help-with-domains/information-for-dns-records.md:86]: @cmcatee-MSFT I have made changes to the document according to the new UI. Also emailed you the screenshots.

**Review by cmcatee-MSFT — APPROVED** [review]

**Pull request merged**